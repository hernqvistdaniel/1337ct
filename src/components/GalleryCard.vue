<script setup lang="ts">
import type { Presentation } from '@/models/presentation';

defineProps<{
  presentation: Presentation
}>()
</script>

<template>
  <p>{{ presentation.name }}</p>
</template>

<style scoped>
</style>
