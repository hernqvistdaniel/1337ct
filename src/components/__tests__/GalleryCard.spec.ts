import { describe, it, expect } from 'vitest'

import { shallowMount } from '@vue/test-utils'
import GalleryCard from '../GalleryCard.vue'

describe('GalleryCard.vue', () => {
  const wrapper = shallowMount(GalleryCard, { props: {} })

  it('renders properly', () => {
    expect(wrapper.html()).toBeTruthy()
  })

  it('logs', () => {
    console.log(wrapper.html())
  })
})