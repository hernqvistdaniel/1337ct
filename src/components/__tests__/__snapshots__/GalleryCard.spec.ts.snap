// Vitest Snapshot v1

exports[`GalleryCard.vue > matches a snapshot 1`] = `
"<div class=\\"card\\" data-v-2b6d1a82=\\"\\">
  <div class=\\"imgContainer\\" data-v-2b6d1a82=\\"\\"><img class=\\"cardImg\\" src=\\"https://i.1337co.de/profile/akan-murat-cimen\\" data-v-2b6d1a82=\\"\\">
    <!--v-if-->
  </div>
  <div class=\\"cardInfo\\" data-v-2b6d1a82=\\"\\">
    <div class=\\"texts\\" data-v-2b6d1a82=\\"\\">
      <p class=\\"cardText\\" data-v-2b6d1a82=\\"\\">Akan Murat Cimen</p>
      <p class=\\"cardText\\" data-v-2b6d1a82=\\"\\">Office: Lund</p>
    </div>
    <div class=\\"iconContainer\\" data-v-2b6d1a82=\\"\\"><a href=\\"http://linkedin.com//in/akanmuratcimen/\\" data-v-2b6d1a82=\\"\\">
        <font-awesome-icon class=\\"icon\\" icon=\\"fas,phone\\" data-v-2b6d1a82=\\"\\"></font-awesome-icon>
      </a><a href=\\"http://github.com/akanmuratcimen\\" data-v-2b6d1a82=\\"\\">
        <font-awesome-icon class=\\"icon\\" icon=\\"fas,gamepad\\" data-v-2b6d1a82=\\"\\"></font-awesome-icon>
      </a>
      <!--v-if-->
    </div>
  </div>
</div>"
`;
